lib/src/alert.dart

```diff
@@ -18,7 +17,9 @@ class Alert extends _WebDriverBase {
    * Throws [WebDriverError] no such alert exception if there isn't currently an
    * alert.
    */
-  Future<Alert> accept() => _post('accept_alert').then((_) => this);
+  Future accept() async {
```

Comment:
No longer.

The return of alert was to make .then() chaining slightly simpler in some cases e.g.:
```
driver.switchTo.alert.then((alert) => alert.dismiss()).then((alert) {
  expect(alert.text, 'this is the alert I wanted');
});
```
rather than:
```
var alert;
driver.switchTo.alert.then((a) {
  alert = a;
  return alert.dismiss();
}).then((_) {
  expect(alert.text, 'this is the alert I wanted');
});
```
but with await it gets even better:
```
var alert = await driver.switchTo.alert;
await alert.dismiss();
expect(alert.text, 'this is the alert I wanted');
```
so returning the Alert doesn't add anything, and was always a little weird anyways.

That being said, I had a slightly different pattern going on in Mouse and Keyboard (and Touch before I deleted it) that was a bit cooler, and allowed proper method chaining. Unfortunately it worked different than the usual method chaining with .. that is more common in Dart, so I decided to kill it off as part of simplifying the APIs.